airbyte-ci/connectors/pipelines/pipelines/publish_pypi.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from textwrap import dedent
+from typing import List
+
+import anyio
+from dagger import ExecError
+from pipelines.bases import ConnectorReport, Step, StepResult, StepStatus
+from pipelines.contexts import PublishConnectorContext
+from pipelines.pipelines import metadata
+
+
+class PublishPyPIConnector(Step):
+    title = "Publish connector to PyPI"
+
+    async def _run(self) -> StepResult:
+        if self.context.connector.language not in ["python", "low-code"]:
+            return self.skip("Only Python connectors can be published to PyPI.")
+
+        try:
+            setup_cfg = dedent(
+                f"""
+                [metadata]
+                name = airbyte-{self.context.connector.technical_name}
+                version = {self.context.connector.version}
+                author = Airbyte
+                author_email = contact@airbyte.io
+            """
+            )
+
+            pypi_stdout = await (
+                self.context.dagger_client.container()
+                .from_("python:3.9-slim")
+                .with_exec(["apt-get", "update"])
+                .with_exec(["apt-get", "install", "-y", "twine"])
+                .with_directory("connector", (await self.context.get_connector_dir(exclude=["dist"])))
+                .with_workdir("connector")
+                .with_exec(["sed", "-i", "/name=/d; /author=/d; /author_email=/d; /version=/d", "setup.py"])
```

Comment:
What's the purpose of this `sed`? Can we provide a ready to consume `setup.py` to the container?